client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -126,6 +153,15 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
       j.ballotManifest.processing.status === FileProcessingStatus.PROCESSED
   ).length
 
+  const initialValues = targetedContests.length
+    ? targetedContests.map(c => ({
+        [c.id]: sampleSizeOptions[c.id] ? sampleSizeOptions[c.id][0].size : '',
```

Comment:
the backend guarantees that the keys of `sampleSizeOptions` will include all the targeted contest ids

Reply:
@jonahkagan I might have been overcautious, but my concern here was that the call to the api to get the sample sizes will not have returned yet on the first run through this component. So `sampleSizeOptions` will only be the default `{}`. We're getting data from two asynchronous sources here, for the targeted contests and the sample sizes, and it needs both in order to create the `initialValues` value. Hence the double-layered ternary and fallback data.